app/src/main/java/org/inspire/breath/activities/ValidateDiarrhoeaActivity.java

```diff
@@ -0,0 +1,84 @@
+package org.inspire.breath.activities;
```

Comment:
The final result of these questions needs to be stored into the DB under the patient profile (see DiarrhoeaTestResult.java)